admin/config/config.yml

```diff
@@ -32,7 +32,7 @@ log_prefix: null # Prefix text to append to log entries
 max_tcp_connections: 100 # max number of inflight tcp connections
 head_sleep_time: 10 # max sleep time between health checks for head node
 node_sleep_time: 10 # max sleep time between health checks for SN/DN nodes
-async_sleep_time: 10 # max sleep time between async task runs
```

Comment:
Why the change in automatic scan frequency? I thought the idea was to allow certain requests to force a scan.

Reply:
The buckeScan task is still running independently, so the sleep is there to not unnecessarily consume cpu.
It might be better to use something like asyncio.Event (see https://docs.python.org/3/library/asyncio-sync.html#asyncio.Event).

Reply:
I'll leave using signals  as a future enhancement since what we have should be good enough for this non-critical path.